dvc/command/logs.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import logging
+
+from dvc.command import completion
+from dvc.command.base import CmdBase
+from dvc.exceptions import DvcException
+
+logger = logging.getLogger(__name__)
+
+
+class CmdLogs(CmdBase):
```

Comment:
We need this command for a scenarion as in https://github.com/iterative/dvclive/issues/7